docs/strategy-advanced.md

```diff
@@ -179,6 +179,35 @@ class AwesomeStrategy(IStrategy):
             return (-0.07 + current_profit)
         return 1
 ```
+#### Custom stoploss using an indicator from dataframe example
+
+Imagine you want to use `custom_stoploss()` to use a trailing indicator like e.g. "ATR"
+
+See: (Storing custom information using DatetimeIndex from `dataframe`
+)[WIP] on how to store the indicator into `custom_info`
+
+``` python
+from freqtrade.persistence import Trade
+
+class AwesomeStrategy(IStrategy):
+
+    # ... populate_* methods
+
+    use_custom_stoploss = True
+
+    def custom_stoploss(self, pair: str, trade: 'Trade', current_time: datetime,
+                        current_rate: float, current_profit: float, **kwargs) -> float:
+
+        result = 1
+        if self.custom_info[pair] is not None and trade is not None:
+            atr = self.custom_info[pair].loc[current_time]['atr']
```

Comment:
using `current_time` directly (like this) will only work in backtesting.

in live / dry-run, it'll be really the current time (like 14:54:44) - not the candle time - so this lookup will fail.

Reply:
Ah, good I made a PR then before going live, sometime karma works out :D

Any idea for a workaround? Only thing I can think right now, would be getting index and of trade.open_date and then walking forward via index until `date >= current_time`. Not sure if this would work in backtesting without looking in into the future (because I would have to use numerical index like this `dataframe[index_starting_at_opendate])

Reply:
Hmm, maybe `.loc[open_date < date < current_time].last()` (pseudocode) would be eleganter. 
Will investigate over the next days if you don't have a better idea into what else I could look into

Reply:
just change how you assign `atr` based on the run-mode.
for live, you can as well use .last (or `.iloc[-1]`) - but you MUST make sure that's not used in backtesting - therefore it's not recommended.